app/src/main/res/layout/filter_bottom_sheet.xml

```diff
@@ -6,7 +6,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:background="@drawable/bottom_sheet_rounded_background"
-    android:elevation="7dp"
+    android:backgroundTint="?attr/colorPrimaryVariant"
```

Comment:
leave this as is, colorPrimaryVariant is for title-like object, not to use as a general background